README.md

```diff
@@ -1 +1,34 @@
-# distributed-task-scheduler
\ No newline at end of file
+# distributed-task-scheduler
+
+## Building and Running Applications
+
+This project contains multiple applications managed using Go workspaces. Below are the instructions to build and run each application.
+
+### Prerequisites
+
+- Ensure you have Go version 1.18 or later installed.
+- Clone the repository and navigate into the project's root directory.
+
+### Building Applications
+
+```bash
+#build worker
+cd worker
+go build -o worker
+
+#build scheduler
+cd mts
```

Comment:
Why did we call this mts?

Reply:
cos I am an idiot, wanted to put dts but put mts but maybe we want to called it scheduler?

Reply:
yes, that seems fine, let's do that then

Reply:
Let's also then use that naming in the diagram as well, let's add a glossary page to the wiki to keep these thing clear as well so we can have a ubiquitous lang

Reply:
Alright, I'll make an issue for the glossary page